src/gsy_e/models/strategy/infinite_bus.py

```diff
@@ -44,15 +44,15 @@ def __init__(self, energy_sell_rate=None, energy_rate_profile=None, energy_buy_r
         # buy
         if all(arg is None for arg in [
                buying_rate_profile, buying_rate_profile_uuid, energy_buy_rate]):
-            energy_buy_rate = ConstSettings.GeneralSettings.DEFAULT_MARKET_MAKER_RATE
+            energy_buy_rate = GlobalConfig.FEED_IN_TARIFF
```

Comment:
I am hesitant to have yet another round for this, but I think that since this parameter is mutable, it is better to follow the same approach as for the MARKET_MAKER_RATE. Essentially to create a new constant named DEFAULT_FEED_IN_TARIFF, and in the GlobalConfig class, to assign it `FEED_IN_TARIFF = ConstSettings.DEFAULT_FEED_IN_TARIFF`. That way the default value will remain unchanged, even if the GlobalConfig.FEED_IN_TARIFF changes.